pkg/virt-controller/services/template_test.go

```diff
@@ -203,6 +203,27 @@ var _ = Describe("Template", func() {
 				Expect(pod.Spec.SecurityContext.SELinuxOptions).ToNot(BeNil())
 				Expect(pod.Spec.SecurityContext.SELinuxOptions.Type).To(Equal("spc_t"))
 			})
+			It("should run under spc_t if network multiqueue is enabled", func() {
+				enabled := true
+				vmi := v1.VirtualMachineInstance{
+					ObjectMeta: metav1.ObjectMeta{
+						Name: "testvmi", Namespace: "default", UID: "1234",
+					},
+					Spec: v1.VirtualMachineInstanceSpec{
+						Volumes: []v1.Volume{},
+						Domain: v1.DomainSpec{
+							Devices: v1.Devices{
+								NetworkInterfaceMultiQueue: &enabled,
+							},
+						},
+					},
+				}
+				pod, err := svc.RenderLaunchManifest(&vmi)
+				Expect(err).ToNot(HaveOccurred())
+				Expect(pod.Spec.SecurityContext).ToNot(BeNil())
+				Expect(pod.Spec.SecurityContext.SELinuxOptions).ToNot(BeNil())
+				Expect(pod.Spec.SecurityContext.SELinuxOptions.Type).To(Equal("spc_t"))
```

Comment:
One thing that I'm concerned about is that the previous test on line 190, sets the spc_t in the fake cluster config, but never removes it. Perhaps you can clear it at the end of the previous test so we'll know that this one is 100% correct.

Reply:
Yes, fixed! This is a problem in other places as well, which I used as a model...